src/integrations/rapier/fluids_pipeline.rs

```diff
@@ -152,108 +153,110 @@ impl<'a> CouplingManager for ColliderCouplingManager<'a> {
         fluids: &mut [Fluid],
         boundaries: &mut BoundarySet,
     ) {
+        let bodies = &self.bodies;
         for (collider, coupling) in &mut self.coupling.entries {
             if let (Some(collider), Some(boundary)) = (
                 self.colliders.get(*collider),
                 boundaries.get_mut(coupling.boundary),
             ) {
                 // Update the boundary's ability to receive forces.
-                let body = self.bodies.get(collider.parent());
-                if let Some(body) = body {
-                    if !body.is_dynamic() {
-                        boundary.forces = None;
-                    } else {
-                        boundary.forces = Some(RwLock::new(Vec::new()));
-                        boundary.clear_forces(true);
+                if let Some(body) = collider.parent().map(|p| bodies.get(p)) {
+                    if let Some(body) = body {
+                        if !body.is_dynamic() {
+                            boundary.forces = None;
+                        } else {
+                            boundary.forces = Some(RwLock::new(Vec::new()));
+                            boundary.clear_forces(true);
+                        }
                     }
-                }
```

Comment:
I think that this closing bracked should be added back. This is what messes up with the indentation afterward.
This probably means that there is a closing bracket to remove further down.